scss/components/_loading-placeholder.scss

```diff
@@ -72,6 +72,15 @@
   }
 }
 
+.civicase__loading-placeholder__date {
+  background-color: $placeholder-grey;
+
+  &::before {
+    border-left-color: darken($placeholder-grey, 7%);
```

Comment:
how do you know the border must be %7 darker?

Reply:
This has been confirmed with Jon. I will add it in the PR and the